### ISSUE TITLE
NVDA cannot separate focus by arrowing when a form field is next to a button

### ISSUE THREAD

**Issue by klmui** [nvaccess/nvda#15128]
### Steps to reproduce:
1. Open Chrome and go to https://codepen.io/kmui123/pen/GRwOBVe
2. Start NVDA
3. Click into the input field.
4. Try moving the focus from the input field to the button while using the arrow keys.

### NVDA logs, crash dumps and other attachments:
N/A

#### NVDA version:
Release 2023.1

#### Windows version:
Version10.0.19044 Build 19044

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
Windows 11

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - same behavior.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
This is intentional behaviour - you are not expected to be able to arrow key out of form controls in focus mode.
In future please explain in more detail the actual and expected behaviour.
